sdk/servicebus/service-bus/samples/typescript/src/receiveMessagesStreaming.ts

```diff
@@ -37,48 +37,56 @@ export async function main() {
+          //
+          // If your handler _does_ throw an error then the message will automatically
+          // be abandoned using receiver.abandonMessage()
+          //
+          // autoComplete can be disabled in the options for subscribe().
+        },
+        processError: async (args: ProcessErrorArgs) => {
+          console.log(`Error from source ${args.errorSource} occurred: `, args.error);
 
-        // the `subscribe() call will not stop trying to receive messages without explicit intervention from you.
-        if (isServiceBusError(args.error)) {
-          switch (args.error.code) {
-            case "MessagingEntityDisabled":
-            case "MessagingEntityNotFound":
-            case "Unauthorized":
-              // It's possible you have a temporary infrastructure change (for instance, the entity being
-              // temporarily disabled). The handler will continue to retry if `close()` is not called on the subscription - it is completely up to you
-              // what is considered fatal for your program.
-              console.log(
-                `An unrecoverable error occurred. Stopping processing. ${args.error.code}`,
-                args.error
-              );
-              await subscription.close();
-              break;
-            case "MessageLockLost":
-              console.log(`Message lock lost for message`, args.error);
-              break;
-            case "ServiceBusy":
-              // choosing an arbitrary amount of time to wait.
-              await delay(1000);
-              break;
+          // the `subscribe() call will not stop trying to receive messages without explicit intervention from you.
+          if (isServiceBusError(args.error)) {
+            switch (args.error.code) {
+              case "MessagingEntityDisabled":
+              case "MessagingEntityNotFound":
+              case "Unauthorized":
+                // It's possible you have a temporary infrastructure change (for instance, the entity being
+                // temporarily disabled). The handler will continue to retry if `close()` is not called on the subscription - it is completely up to you
+                // what is considered fatal for your program.
+                console.log(
+                  `An unrecoverable error occurred. Stopping processing. ${args.error.code}`,
+                  args.error
+                );
+                await subscription.close();
+                break;
+              case "MessageLockLost":
+                console.log(`Message lock lost for message`, args.error);
+                break;
+              case "ServiceBusy":
+                // choosing an arbitrary amount of time to wait.
+                await delay(1000);
+                break;
+            }
           }
         }
+      },
+      {
+        // Indicates whether the receiver.completeMessage() should automatically be called for your message by the sdk
```

Comment:
A few lines above inside the processMessage callback, you will see comments reflecting the same message. I understand that we are doing this as a reaction to user feedback on v1 samples which did not have these details. 

I believe the processMessage callback would be a better place to communicate this as that is the part that the user would first change to suit their needs.